src/emerge.cpp

```diff
@@ -129,6 +131,7 @@ EmergeParams::EmergeParams(EmergeManager *parent, const BiomeManager *biomemgr,
 	biomemgr(biomemgr->clone()), oremgr(oremgr->clone()),
 	decomgr(decomgr->clone()), schemmgr(schemmgr->clone())
 {
+	this->biomegen = biomegen->clone(this->biomemgr);
```

Comment:
Where is it feed? If not explicitly required (e.g. handled automatically), a comment would be appropriate here.

Reply:
https://github.com/minetest/minetest/blob/ba63b2bb3947128db020b4a96b56d3f9074a5f65/src/emerge.cpp#L116